### ISSUE TITLE
output filename with replacement seems not working

### ISSUE THREAD

**Issue by llaville** [box-project/box2#75]
For the last report today.

I've try to use the feature used by your file 
https://github.com/kherge-old/Box/blob/2.0/box.json.dist#L35

But it seems not working for me. Can you confirm it's possible ?

Thanks in advance
Laurent

**Comment by llaville** [comment]
I've used both syntax 

```
"git-commit": "git_commit",
"git-version": "package_version",
"output": "phpreflect-@git-version@.phar",
```

and

```
"git-commit": "git_commit",
"git-version": "package_version",
"output": "phpreflect-@package_version@.phar",
```

But I always got file not renamed.

**Comment by ghost** [comment]
Is the project you're working on available somewhere? If it's okay, I'd like to test it directly against your code. I know this works for Box, since I do get versioned output files.

**Comment by llaville** [comment]
Yes, of course you can.
The project is on the branch v2 here https://github.com/llaville/php-reflect/tree/v2
with the box.json file (where I've removed "output" renaming option)

**Comment by ghost** [comment]
Okay, I was able to repeat the problem, even in Box. I don't know what's changed but I'll try and fix it.

**Comment by ghost** [comment]
Note: This problem only happens with the phar version of Box, but not with the Composer installed version.

**Comment by ghost** [comment]
Okay, it's fixed! Use Box 2.4.4.

**Comment by llaville** [comment]
I confirm, it works fine. Thanks :)
